cypress/plugins/excludeByVersionTags.js

```diff
@@ -87,4 +88,15 @@ const getExcludedTags = (v) => {
     return excludeTags
 }
 
-module.exports = { getExcludedTags }
+const excludeByVersionTags = (on, config) => {
+    const excludedTags = getExcludedTags(config.env.dhis2InstanceVersion)
+
+    console.log('instanceVersion', config.env.dhis2InstanceVersion)
+    console.log('tags to exclude from testing', excludedTags)
+
+    config.env.CYPRESS_EXCLUDE_TAGS = excludedTags.join(',')
+
+    on('file:preprocessor', tagify(config))
+}
```

Comment:
@jenniferarnesen: this logic was previously defined in `cypress/plugins/index.js` which is now removed. Most of what was previously there is now in `cypress.config.js`, but I didn't think this logic belonged in there. And since this is technically a plugin, moving it here seemed like the right thing to do.

Reply:
Fine with me as long as it still works :)